rholang/src/test/scala/coop/rchain/rholang/interpreter/ReduceSpec.scala

```diff
@@ -42,8 +43,9 @@ trait PersistentStoreTester {
 }
 
 class ReduceSpec extends FlatSpec with Matchers with PersistentStoreTester {
-
   "evalExpr" should "handle simple addition" in {
+    val errorLog                                         = new Runtime.ErrorLog()
```

Comment:
Any reason to not just make this `implicit val errorLog` and save a line?

Reply:
done everywhere